website/src/pages/specs/activitypub.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: ~/layouts/MdLayout.astro
+title: ActivityPub conformance
+---
+
+Apart from a few side-effects which would be quick to implement if need arised, ActivityPods is 100% compatible with ActivityPub, and supports both client-to-server and server-to-server interactions.
+
+### Outbox
+
+### Inbox
+
+### Create/Update/Delete
+
+It's possible to create, update or delete any kind of objects. Objects are automatically stored in the corresponding containers.
+
+### Follow
+
+Every ActivityStreams actor is automatically attached `as:followers` and `as:following` collections. Whenever a Follow activity is detected, it will add
```

Comment:
?

Reply:
Sentence not finished. This is still very much WIP.